doc/usage.rst

```diff
@@ -97,29 +97,34 @@ directory:
     Entering directory '/home/jdimino/code/dune'
     ...
 
-More precisely, it will choose the outermost ancestor directory containing a
-``dune-workspace`` file as root. For instance if you are in
-``/home/me/code/myproject/src``, then dune will look for all these files in
-order:
-
--  ``/dune-workspace``
--  ``/home/dune-workspace``
--  ``/home/me/dune-workspace``
--  ``/home/me/code/dune-workspace``
--  ``/home/me/code/myproject/dune-workspace``
--  ``/home/me/code/myproject/src/dune-workspace``
-
-The first entry to match in this list will determine the root. In
-practice this means that if you nest your workspaces, dune will
-always use the outermost one.
-
-In addition to determining the root, ``dune`` will read this file
-to setup the configuration of the workspace unless the ``--workspace``
-command line option is used. See the section :ref:`dune-workspace`
-for the syntax of this file.
-
-The ``Entering directory`` message can be suppressed with the
-``--no-print-directory`` command line option (as in GNU make).
+This message can be suppressed with the ``--no-print-directory``
+command line option (as in GNU make).
+
+Dune requires at least one ``dune-workspace`` file or ``dune-project``
```

Comment:
Dune requires at least one ``dune-workspace`` file or ``dune-project``
file to be present in the current directory (or a parent one) in order
to operate.

Dune requires at least one ``dune-workspace`` file or ``dune-project``
file to be present in the current (or parent) directory in order
to operate.

I'd suggest one of these, but it's also correct the way it is.

Reply:
I updated the text following our discussion on slack